test/uk/gov/hmrc/transitmovementsauditing/routing/VersionedRoutingControllerSpec.scala

```diff
@@ -62,7 +62,7 @@ class VersionedRoutingControllerSpec extends AnyWordSpec with Matchers with Mock
   "post" should {
     "call the transitional controller when APIVersion non 'final' value has been sent" in new Setup {
```

Comment:
Transitional tests can be removed, you have removed the functionality in the controller already